packages/botonic-plugin-contentful/src/tools/l10n/csv-import.ts

```diff
@@ -109,18 +164,38 @@ export class RecordFixer {
     this.record.to = replaceAll(this.record.to, '_x000D_', '')
   }
 
+  private fixStringArraySeparators(stringArray: string): string {
+    for (const c of ',.?') {
+      if (stringArray.indexOf(c) >= 0) {
+        this.complain(`... Replacing '${c}' with ;`)
+        stringArray = replaceAll(stringArray, c, ';')
+      }
+    }
+    return stringArray
+  }
+
   checkKeywordLen(): void {
-    if (this.field.valueType == ContentFieldValueType.STRING_ARRAY) {
+    if (this.field.fieldType == ContentFieldType.KEYWORDS) {
       const fromLen: number = this.field.parse(this.record.from).length
       const toLen: number = this.field.parse(this.record.to).length
-      if (toLen != fromLen) {
+      if (toLen != fromLen || fromLen == 1) {
```

Comment:
We can not have only one keyword?

Reply:
it's just a warning. when there's only only, it's probably that we're using a wrong separator